packages/tree-widget/src/components/header/TreeHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
```

Comment:
copyright header

Reply:
i should look into adding the linter to the monorepo

Reply:
Yeah, I've been working on trying to adapt what kevin has in the itwin-viewer to be a little bit more broad for use in a lot of our repos.  I can send you the PR I have in progress for iModel.js

Reply:
Forgot :) fixed